packages/lit-auth-client/src/lib/providers/BaseProvider.ts

```diff
@@ -175,27 +175,22 @@ export abstract class BaseProvider {
     }
 
     const litContracts = new LitContracts({
-      randomPrivatekey: true,
-      network: this.litNodeClient.config.litNetwork,
+      signer: new ethers.Wallet(ethers.Wallet.createRandom().privateKey, new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)),
+      network: LIT_NETWORK.DatilDev,
+      rpc: LIT_RPC.CHRONICLE_YELLOWSTONE
     });
-    try {
-      await litContracts.connect();
-    } catch (err) {
-      throw new UnknownError(
-        {
-          cause: err,
-        },
-        'Unable to connect to LitContracts'
-      );
-    }
 
+    console.log("🔄 Connecting to Lit Contracts...");
+    await litContracts.connect();
+    console.log("✅ Connected to Lit Contracts");
+
+    let pkps: IRelayPKP[] = [];
     try {
       const pkpPermissions = litContracts.pkpPermissionsContract;
       const tokenIds = await pkpPermissions.read.getTokenIdsForAuthMethod(
-        authMethodType,
-        authMethodId
+        authMethod.authMethodType,
+        await googleProvider.getAuthMethodId(authMethod)
```

Comment:
This doesn't need to be changed, and `googleProvider` doesn't seem to ever be defined
```suggestion
        authMethodType,
        authMethodId
```